src/groups/bmq/bmqpi/bmqpi_dttracer.h

```diff
@@ -56,6 +57,23 @@ class DTTracer {
         const bsl::shared_ptr<DTSpan>& parent,
         const bsl::string_view&        operation,
         const DTSpan::Baggage&         baggage = DTSpan::Baggage()) const = 0;
+
+    /// Serialize the underlying context for a specified `DTSpan` and return
+    /// the bit stream as `out`.  Return 0 on success, or a non-zero error code
+    /// on error.
+    virtual int serializeSpan(bsl::vector<unsigned char>*    out,
+                              const bsl::shared_ptr<DTSpan>& dtSpan) const = 0;
+
+    /// Deserialize the specified bit stream `in` into a span context.
+    /// Create a child span and pass into `out` based on the deserialized
+    /// parent span context, the given `operation`, and an optional key-value
+    /// tags `baggage`.  Return 0 on success, or a non-zero error code on
+    /// error.
```

Comment:
```
/// Deserialize the specified `in` buffer as a span, and then construct a new `DTSpan` in `out`, representing `operation` as a child of the deserialized span and having the key-value tags defined by `baggage`.
```

Reply:
I think we can do better with the parameter names, too.  `out` and `in` don't tell me much.  `child` and `buffer` tell me more.